platform/darwin/src/MGLGeometry.h

```diff
@@ -45,6 +45,18 @@ typedef struct MGLCoordinateBounds {
     CLLocationCoordinate2D ne;
 } MGLCoordinateBounds;
 
+typedef struct MGLCoordinateQuad {
```

Comment:
This struct needs a documentation comment in order for the struct to appear in jazzy documentation.